public/app/core/components/DynamicImports/SafeDynamicImport.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Loadable from 'react-loadable';
+import { css } from 'emotion';
+import { LoadingChunkPlaceHolder } from './LoadingChunkPlaceHolder';
+import { ErrorLoadingChunk } from './ErrorLoadingChunk';
+
+function getAlertPageStyle(): string {
```

Comment:
stylesFactory please :)

Reply:
I think I'll move this into ErrorLoadingChunk as well